.eslintrc

```diff
@@ -38,7 +38,7 @@ rules:
     - 2
     - always
   space-before-function-paren:
-    - 2
```

Comment:
To this, and the indent rule change above: why change the code style conventions at this point? I'd just keep it consistent with what's already there.